challengeMk2/DataBase/ILocalDataService.cs

```diff
@@ -1,20 +1,21 @@
 using System;
 using System.Collections.Generic;
+using System.Threading.Tasks;
 using ChallengeMk2.Models;
 
 namespace ChallengeMk2.DataBase
 {
     public interface ILocalDataService
     {
-        void Initialize();
+        Task Initialize();
 
-        List<StarSystemDbItem> GetFullDb();
+        Task<List<StarSystemDbItem>> GetFullDb();

```

Comment:
`GetFullDb` is misleading. I would rename it to something like `GetAll`.

Reply:
You're totally right ! furthermore, i call that method from an object called Database...So Database.GetAllAsync() will be perfect.